Automattic/wp-calypso
# Swapping in new Facebook card content on Marketing Tools

#### Changes proposed in this Pull Request

* Updating the Facebook card on Marketing Tools
* Additional context in pbBQWj-Hz-p2
* This will show EN only so doesn't need to wait for translations.

Here's a screenshot of the updated card:
<img width="530" alt="Screen Shot 2021-03-12 at 10 07 13 AM" src="https://user-images.githubusercontent.com/35781181/110966825-b218dd00-8323-11eb-8bdb-1f7e5eaf6660.png">


#### Testing instructions
* Checkout this PR and spinup Calypso.
* Navigate to marketing/tools/SITESLUG
* Confirm that the Facebook slug reflects the changes outlines in the P2 post above.
